library/src/scala/caps/package.scala

```diff
@@ -132,6 +133,12 @@ final class use extends annotation.StaticAnnotation
 @deprecated
 sealed trait Exists extends Capability
 
+/** A wrapper that strips all covariant capture sets from Mutable types in the
+ *  result of pure operation `op`, turning them into immutable types.
+ */
+@experimental
+def immutable[T](op: -> T): T = op
```

Comment:
to me this sounds like a block where mutability should be prevented *on the inside*, (i.e. like its some pure scope). Alternatively this has an effect similar to "freeze" in other languages (i.e the result is now immutable).

Reply:
Yes, I was unsure as well. I also thought of `freeze` before. Do you know which other languages have a construct like this?